DALOptimizer/Program.cs

```diff
@@ -28,8 +28,10 @@ public static void Main(string[] args)
                 MessageBox.Show("Please Select one file ");
                 return;
             }
+            
 
             MatchInvocation Mr = new MatchInvocation();
```

Comment:
MatchInvocation matchInvocation = new MatchInvocation();
You can use naming convention like this.